docs/Tutorials/Start-EthSigner.md

```diff
@@ -22,14 +22,15 @@ For file-based signing, EthSigner requires a V3 Keystore key file and a password
 
 ## Start Besu
 
-[Start Besu](https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Starting-node/) with the
-[`--rpc-http-port`](https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#rpc-http-port)
-option set to `8590`.
+[Start Besu](https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Starting-node/), setting the:
+
+* [`--rpc-http-port`](https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#rpc-http-port) option to `8590`
+* [`--data-path`](https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#rpc-data-path) option to an existing directory.
```

Comment:
is the "existing directory" constraint important to prevent a failure? If so, is it worth creating a ticket to ask to add a test and abort in case the directory doesn't already exist?

Reply:
Besu already aborts with a error if the directory doesn't exist - I've raised a JI to improve the error message.

Reply:
So it seems the current behaviour of when Besu does and doesn't create the extra directories isn't entirely clear - I've updated the JI for this https://jira.hyperledger.org/browse/BESU-188 

In terms of this PR - I think something like makes it clear you need to update it: 
--data-path option to an appropriate directory.

Reply:
Updated the --data-path option to "appropriate" directory.